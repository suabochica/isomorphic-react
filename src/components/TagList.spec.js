import React from 'react';
import renderer from 'react-test-renderer'

import TagList from './TagsList'

describe("The Tag List snapshot", () => {
    it("should render as expected", () => {
        const tree = renderer
            .create(<TagList tags={[`css`, `html`, `go`]} />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})
