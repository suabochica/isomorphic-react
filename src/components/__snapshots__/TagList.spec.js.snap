// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Tag List snapshot should render as expected 1`] = `
<div>
  <code>
     
    css
     
  </code>
  <code>
     
    html
     
  </code>
  <code>
     
    go
     
  </code>
</div>
`;
